File: PieGallery/app/src/main/java/com/deadely/piegallery/view/main/presenter/MainPresenter.java
package com.deadely.piegallery.view.main.presenter;

import com.deadely.piegallery.base.BasePresenter;
import com.deadely.piegallery.view.main.IMainContract;

import javax.inject.Inject;


public class MainPresenter extends BasePresenter<IMainContract.View> implements IMainContract.Presenter {
    @Inject
    public MainPresenter() {
    }
}

File: itlen/app/src/main/java/com/deadely/itl_en/ui/main/presenter/MainPresenter.kt
package com.deadely.itl_en.ui.main.presenter

import com.deadely.itl_en.base.BasePresenter
import com.deadely.itl_en.ui.main.IMainContract
import javax.inject.Inject

class MainPresenter : BasePresenter<IMainContract.View>, IMainContract.Presenter {
    @Inject
    constructor()
}